.github/workflows/refresh-cache.yml

```diff
@@ -12,7 +12,8 @@ jobs:
     steps:
       - uses: actions/checkout@v3
         with:
-          sparse-checkout: '.'
```

Comment:
does this still check out the root directory with`package.json` required for the Node.js deps caching action to work?

Reply:
it uses "cone mode" for sparse checkout which includes all of the files directly under each directory along the way (`.`, `meta`, `meta/remark`)

https://github.com/cl8n/osu-wiki/actions/runs/6679162109/job/18151011485